sysutils/tmux/Portfile

```diff
@@ -3,14 +3,14 @@
 PortSystem      1.0
 PortGroup       github 1.0
 
-github.setup    tmux tmux 3.1b
+github.setup    tmux tmux 3.1c
 if {${subport} eq ${name}} {
     revision        0
     conflicts       tmux-devel
 }
 subport tmux-devel {
-    github.setup    tmux tmux a08f1c8c59787408529463c985929e5ae1c67211
-    version         20200504-[string range ${github.version} 0 6]
+    github.setup    tmux tmux d0640609044c06ffa5aeb7a509965ce3f0b1a412
+    version         20201030-[string range ${github.version} 0 6]
```

Comment:
As you also update the devel port here as well as the main one, please either mention this in the commit message, or ideal do it in a separate commit to the primary port update.

Reply:
This has never been an issue before. I won't create 2 PRs for that and adding 2 commits are also not wanted by some. So, you can either merge this, make the changes yourself, or look for a different maintainer.
I'm sorry, but I seriously don't have time for this kind of nitpicking.

Reply:
I am not asking for two PRs. I am simply asking if you modify two ports in a single commit, to mention both in the commit message.

Reply:
B.t.w. This requirement to list any modified ports is clearly outlined in the commit guidelines, which you indicated you had followed...